c2corg_ui/static/js/documentediting.js

```diff
@@ -559,6 +542,52 @@ app.DocumentEditingController.prototype.pushToArray = function(object, property,
   }
 }
 
+app.DocumentEditingController.prototype.pushToArray = function(object, property, value, event) {
+  var pushed = app.utils.pushToArray(object, property, value);
+  var checkbox = $(event.currentTarget).find('input');
+
+  if (pushed) {
+    checkbox.prop('checked', true);
+  } else {
+    checkbox.prop('checked', false);
+  }
+}
+
+app.DocumentEditingController.prototype.pushToArray = function(object, property, value, event) {
```

Comment:
even 3 times! surely something went wrong when resolving conflicts. Thank's for telling.